src/data/mfsexports.cfg

```diff
@@ -41,7 +41,7 @@
 #  4h30M = 4 hours and 30 minutes (time units are case insensitive)
 #  12w = 12 weeks
 #  86400 = 86400 seconds = 1 day
-#  11d13h46m40s = 1000000 seconds (defined in a little strage way as 11 days, 13 hours, 46 minutes and 40 seconds)
+#  11d13h46m40s = 1000000 seconds (defined in a little strange way as 11 days, 13 hours, 46 minutes and 40 seconds)
```

Comment:
One more typo in line 23 `auxilinary` instead of `auxiliary`. I recommend `:set spell spelllang=en_us`. :+1: